release_notes/ocp_3_4_release_notes.adoc

```diff
@@ -1702,3 +1702,194 @@ openshift3/registry-console:3.4-30
 To upgrade an existing {product-title} 3.3 or 3.4 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-6-rhba-2017-3049-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 3.5 or 3.6 cluster to this latest
+release, use the automated upgrade playbook. See
+xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
```

Comment:
I think this was a copy/paste from 3.6 content, needs version tweaks.

Reply:
Oh actually this looks like it shouldn't be here at all maybe? From the previous section?